meson_options.txt

```diff
@@ -44,7 +44,7 @@ option('bluez5_dun', type: 'boolean', value: false, description: 'enable Bluez5
 
 # configuration plugins
 option('config_plugins_default', type: 'string', value: '', description: 'Default configuration option for main.plugins setting, used as fallback if the configuration option is unset')
-option('config_plugin_ibft', type: 'boolean', value: true, description: 'enable ibft configuration plugin')
+option('ibft',     type: 'boolean', value: false, description: 'enable ibft configuration plugin')
```

Comment:
There are a couple whitespaces in front of type that would be nice to remove.

Reply:
the white space is intentional, to vertically align with the neighbouring lines, that do something similar.